PathPlanning/DynamicMovementPrimitives/dynamic_movement_primitives.py

```diff
@@ -0,0 +1,260 @@
+"""
```

Comment:
Please add a one-line comment to describe this example here.

Reply:
Added one to the example_DMP() function toward the bottom of the file

Reply:
Can you add a one-line comment to describe this file here? Like: "Dynamic Movement Primitives examples for robotics".